maps/southern_cross/southern_cross-3.dmm

```diff
@@ -12506,7 +12506,7 @@
 	desc = "Used for watching the Particle Accelerator chamber.";
 	layer = 4;
 	name = "Particle Accelerator Chamber Telescreen";
-	network = list("PA Room");
+	network = list("PA  Room");

```

Comment:
This appears to of done some strange things in various places to unrelated parts of the map.

Reply:
that is just one example. More visible in the diffs. lots of double-spaced strings

Reply:
I manually removed them, saving it again in SDMM just adds even MORE spaces.